app/feature/feature-home/src/main/kotlin/com/hedvig/android/feature/home/home/ui/HomeDestination.kt

```diff
@@ -112,6 +112,7 @@ internal fun HomeDestination(
   openUrl: (String) -> Unit,
   openAppSettings: () -> Unit,
   navigateToMissingInfo: (String) -> Unit,
+  openChat: () -> Unit,
```

Comment:
I see `onStartChat` was already a function here for HomeDestination, which was used from the chat button top-right I suppose.
These two lambdas just do the exact same thing here right? If yes I'd say just drop this one completely, and pass `onStartChat` instead to the member reminder cards composable